Tasks/Common/find-files-legacy/findfiles.legacy.ts

```diff
@@ -44,7 +44,7 @@ function extractPattern(patterns: string[], cwd: string): any {
     let excludePatterns: string[] = [];
 
     patterns.forEach(p => {
-        p.split(',').forEach(inputFilter => {
+        p.split(';').forEach(inputFilter => {
```

Comment:
this is being used by our tasks only, which use ';' as delimiter between patterns. So, changed to ';'